pkg/client/testdata/single-node-selector.golden

```diff
@@ -71,12 +71,12 @@ data:
       env:
       - name: CHROOT_DIR
         value: /node
-      - name: RESULTS_DIR
-        value: /tmp/results
       - name: NODE_NAME
         valueFrom:
           fieldRef:
             fieldPath: spec.nodeName
+      - name: RESULTS_DIR
```

Comment:
Moving these into alphabetical order is what I would have expected. Not exactly sure which part caused this to change but IMO its just a good thing.